crates/codegen/language/definition/src/model/manifest.rs

```diff
@@ -144,3 +189,36 @@ pub enum BuiltInLabel {
     LeadingTrivia,
     TrailingTrivia,
 }
+
```

Comment:
note to self: should rename/change the older `BuiltInLabel` as it is now confusing to read/use with the new "BuiltIn" types .. Maybe I can use something like `PredefinedLabels` ..